### PR TITLE
New SOE guidance page with diagrams

### PR DESCRIPTION
**Pull request by kd-ods**
# Overview

- What does this pull request do?

Adds a number of docs files. A new docs page of state-owned enterprise (SOE) modelling guidance and 7 associated diagrams.

- How can a reviewer test or examine your changes?

Read the guidance and make sure it (a) gives good modelling advice and (b) reads ok.

- Who is best placed to review it?

@ScatteredInk is best able to review this.

(Closes/Relates to) issue: #363 

## Translations

- [x] New or edited strings appearing as a result of this PR will be picked up for translation
- [x] I've notified the translation coordinator of any new strings that will need
      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

- [x] I've thought about how and when this needs to be released. See:
      https://openownership.github.io/bods-dev-handbook/standard_releases.html      
- [ ] I've updated the changelog, if necessary.
- n/a - I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
- n/a - I've added or edited any other related documentation. See: https://github.com/openownership/bods-dev-handbook/blob/master/style_guide.md

### CODE REVIEW

**Comment by kd-ods** [comment]
After we've merged #403, I'll merge master into this branch, then update the changelog to reference this new guidance page.

**Comment by kd-ods** [comment]
@ScatteredInk - this is the docs PR for the SOE work. (Basically one new docs page with diagrams.) If you could review it over the next working day or two, that'd be great.

File: docs/schema/guidance/repr-state-owned-enterprises.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Overview
+------------------------
+
+The definition of 'state-owned enterprise' (SOE) can change according to jurisdiction or domain. The Beneficial Ownership Data Standard (BODS) does not therefore rely on, or support, any one definition. Instead, BODS enables the interests between a state (or state body) and an enterprise to be represented. This requires the state itself (or state body) to be represented by an Entity statement.
+
+In particular, the following properties and values SHOULD be used to represent information about enterprises and their links to states:
+
+* ``entityType`` SHOULD be set to 'state' or 'stateBody', and ``jurisdiction`` SHOULD be used to represent the particular state.
+* ``entitySubtype`` SHOULD be used to refine the category of a given state body (as a 'government department', 'state agency' or 'other').
+* ``formedByStatute`` SHOULD be used to give details of the legislation under which an enterprise was formed.
```
Comment (ScatteredInk) [inline docs/schema/guidance/repr-state-owned-enterprises.rst:20]: I think we should remove the word 'enterprise' here and use entity as elsewhere.

I'm also not sure that this is a SHOULD unless we clarify that this only applies to entities that are explicitly formed by legislation - we don't want people hunting around for irrelevant legislation to fill in this field.

**Review by ScatteredInk — CHANGES_REQUESTED** [review]
Thanks @kd-ods - this looks really good.

I think the scenarios look great but I think it might be useful to have a framing sentence before or after to clarify that an SOE may need to combine multiple interest types in order to fully represent control. Scenario 6 is an example of this - I think it's unlikely that "control by legal framework" is the beginning and end of control in many cases.

File: docs/schema/guidance/repr-state-owned-enterprises.rst (hunk 1 above)
Comment (kd-ods) [reply docs/schema/guidance/repr-state-owned-enterprises.rst]: I've edited that line now... to a MAY with 'entity' instead of 'enterprise'

**Review by ScatteredInk — CHANGES_REQUESTED** [review]

**Review by ScatteredInk — APPROVED** [review]

**Pull request merged**